src/exception/Exception.cpp

```diff
@@ -0,0 +1,27 @@
+/***********************************************************************/
+/*  Copyright 2015 ToPS                                                */
+/*                                                                     */
+/*  This program is free software; you can redistribute it and/or      */
+/*  modify it under the terms of the GNU  General Public License as    */
+/*  published by the Free Software Foundation; either version 3 of     */
+/*  the License, or (at your option) any later version.                */
+/*                                                                     */
+/*  This program is distributed in the hope that it will be useful,    */
+/*  but WITHOUT ANY WARRANTY; without even the implied warranty of     */
+/*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the      */
+/*  GNU General Public License for more details.                       */
+/*                                                                     */
+/*  You should have received a copy of the GNU General Public License  */
+/*  along with this program; if not, write to the Free Software        */
+/*  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,         */
+/*  MA 02110-1301, USA.                                                */
+/***********************************************************************/
+
+// Internal headers
+#include "Exception.hpp"
```

Comment:
@ademar111190, use the following standard:

```c++
// Interface header
#include "exception/Exception.hpp"
```

I've changed the way `make` generates this files and all `.cpp` files comply with it. The `// Interface header` comment is aimed **only** to the header that is being implemented by this source file. All other `.hpp` go under  `// Standard headers` or `// External headers`.